.travis.yml

```diff
@@ -37,7 +37,10 @@ before_script:
   - echo "zend_extension=xdebug.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
 
 script:
-  - vendor/bin/phpunit -c phpunit.xml.dist --coverage-clover build/logs/clover.xml
+  - vendor/bin/phpunit -c phpunit.xml
 
 after_script:
   - coveralls -v
+
+after_success:
```

Comment:
The code coverage is executed by PHPUnit. The after_script and `coveralls` command is usefull to send data to the inspector. Don't need to add an after_success command to travis